assets/js/paywalled-content.js

```diff
@@ -128,6 +125,7 @@ jQuery(document).ready(function($) {
                     const postIdCopy = configData.postId;
 
                     function tryValidateUnlock(attempt) {
+                        console.log('Unlock validation attempt ' + attempt + (attempt === 5 ? ' (final)' : '') + '...');
```

Comment:
This is on the client right? We don't want to be console.logging in the user's browser.